### PR TITLE
[Docs]update readme

### PR DESCRIPTION
**Pull request by zhouyun-zoe**
* Replace Appchain with CITA
* Replace Neuron with Cyton

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+![](https://img.shields.io/badge/made%20for-CITA-blue.svg)
 
-# Neuron Wallet (Android)
+# Cyton Wallet (Android)
 
 Overview
 ===============
 
-Neuron is an open source blockchain wallet which supports Ethereum and [AppChain](https://docs.nervos.org/#/). It supports most tokens of Ethereum and [AppChain](https://docs.nervos.org/#/), such as ETH, ERC20, ERC721, and also supports most kinds of DApps of Ethereum and [AppChain](https://docs.nervos.org/#/) , such as cryptokitties, Fomo3D, 0xproject...
+Cyton is an open source blockchain wallet which supports Ethereum and [CITA]. It supports most tokens of Ethereum and [CITA], such as ETH, ERC20, ERC721, and also supports most kinds of DApps of Ethereum and [CITA] , such as cryptokitties, Fomo3D, 0xproject...
```
Comment (duanyytop) [inline README.md]: If we does have CITA doc website, I suggest to replace [CITA] with CITA

Comment (ashchan) [reply README.md]: @duanyytop Please do some editing. We can leave `CITA` as blank but add the doc link later when that's updated. But do change `[CITA]` to `CITA` as `[]` is for markdown `[]()` link.

Actually I guess we can just leave a brief introduction (one paragraph?) to Cyton here. This means we have very little marketing description above `Getting Started` section. It doesn't make much sense to have this long description across the doc site and iOS/Android repo README.

Comment (ashchan) [reply README.md]: Also, when linking to CITA doc in the future, we should only link the first occurrence of CITA, not having a lot of the same link all over the place.

Comment (duanyytop) [reply README.md]: OK. I know and I agree with you to show brief introduction of wallet and CITA.

**Comment by ashchan** [comment]
@xiaoluSF Could you take a look at this and update the README for iOS as well?

**Comment by ashchan** [comment]
@duanyytop One thing though: the `master`/`develop` and RC branches are a little bit confusing. For example, the above doc change should have gone to `develop` and `0.7.1 RC` as per our release flow.

**Comment by ashchan** [comment]
I think we should set `develop` branch to default as other repos. How you you think, @duanyytop?

**Comment by duanyytop** [comment]
OK. I agree

**Comment by duanyytop** [comment]
And this PR should be set merge to develop

**Comment by ashchan** [comment]
It's not a hot fix because we're releasing tomorrow :-). We can cherry pick that to the release branch.

**Comment by xiaoluSF** [comment]
@ashchan OK

**Comment by duanyytop** [comment]
I have cherry pick that to develop branch and rebase develop branch to release branch

**Pull request merged**